march_state_machine/src/march_state_machine/gaits/tilted_path_left_knee_bend_sm.py

```diff
@@ -0,0 +1,28 @@
+import smach
+
+from march_state_machine.state_machines.step_state_machine import StepStateMachine
+from march_state_machine.states.idle_state import IdleState
+
+
+def create():
+    sm_tilted_path_left_knee_bend = smach.StateMachine(outcomes=['succeeded', 'preempted', 'failed'])
+    sm_tilted_path_left_knee_bend .register_io_keys(['sounds'])
+    with sm_tilted_path_left_knee_bend:
+        smach.StateMachine.add('GAIT TP LEFT KNEE BEND', StepStateMachine('tilted_path_left_knee_bend',
+                               subgaits=['right_open']),
+                               transitions={'succeeded': 'STANDING TP LEFT STRAIGHT', 'failed': 'failed'})
```

Comment:
`failed` transition can be removed, since it can be implied.
```suggestion
                               transitions={'succeeded': 'STANDING TP LEFT STRAIGHT'})
```